CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ### vNext
+- `apollo-engine-reporting`: Add new EngineReportingOption `enforcePrivateVariables`, and begin deprecating `privateVariables` [PR #2847](https://github.com/apollographql/apollo-server/pull/2847)
```

Comment:
I think the changelog needs to call out the backwards-incompatible nature of this change. Something like 

>**BREAKING CHANGE**: By default, send no GraphQL variable values to Apollo's servers instead of sending all variable values. Use the new EngineReportingOption `enforcePrivateVariables` to send some or all variable values, possibly after transforming them. The `privateVariables` option is now deprecated.

Reply:
I believe `privateVariables` previously also defaulted to blocklisting all variables (set in monorepo). Would this still be considered a breaking change?

Reply:
Are you sure? It certainly didn't a while ago. It might have changed while I wasn't paying attention. What do you mean "set in monorepo"?

Reply:
This was what I was referring to: https://github.com/mdg-private/monorepo/pull/2021

But I realize Adam might have meant that this was set internally. My bad!

Reply:
That change means that in the particular instance of us using apollo-engine-reporting *ourselves* in a particular apollo-server instance that we run, we turned on privateVariables. It was not a change that affected other users' apollo-servers.

Reply:
I see -- this would be a bigger change then I'd otherwise expected then. Not sure if defaulting to the blocklist makes sense then...

Reply:
Resolved -- going to change default settings!